### PR TITLE
Use Kafka official docker image, without zookeeper

### LINKED ISSUE #9076: Kafka: use official image, without zookeeper
With release 3.7, Apache now has [its own official image](https://hub.docker.com/r/apache/kafka).
We should switch to it, and at the same time switch to kraft mode, that don't require running zooker anymore.

### PR DESCRIPTION
**Pull request by murdos**
Fixes #9076

### CODE REVIEW

File: src/main/resources/generator/server/springboot/broker/kafka/kafka.yml.mustache [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
       - '127.0.0.1:9092:9092'
     environment:
-      KAFKA_BROKER_ID: 1
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:29092,PLAINTEXT_HOST://localhost:9092
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
-      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+      KAFKA_NODE_ID: 1
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: 'CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT'
+      KAFKA_ADVERTISED_LISTENERS: 'PLAINTEXT_HOST://localhost:9092,PLAINTEXT://broker:19092'
```
Comment (murdos) [inline src/main/resources/generator/server/springboot/broker/kafka/kafka.yml.mustache:12]:
```suggestion
      KAFKA_ADVERTISED_LISTENERS: 'PLAINTEXT_HOST://localhost:9092,PLAINTEXT://kafka:19092'
```

File: src/main/resources/generator/server/springboot/broker/kafka/kafka.yml.mustache [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+      KAFKA_PROCESS_ROLES: 'broker,controller'
+      KAFKA_CONTROLLER_QUORUM_VOTERS: '1@kafka:29093'
+      KAFKA_LISTENERS: 'CONTROLLER://:29093,PLAINTEXT_HOST://:9092,PLAINTEXT://:19092'
```
Comment (murdos) [inline src/main/resources/generator/server/springboot/broker/kafka/kafka.yml.mustache:15]: Change all instances of 19092 to 29092 to keep compatibility with akhq
```suggestion
      KAFKA_LISTENERS: 'CONTROLLER://:29093,PLAINTEXT_HOST://:9092,PLAINTEXT://:29092'
```

**Comment by pascalgrimaud** [comment]
@murdos : is it ready?

**Comment by murdos** [comment]
> @murdos : is it ready?

Yes :)

**Pull request merged**